src/myuw-help.js

```diff
@@ -70,12 +76,23 @@ class MyUWHelp extends HTMLElement {
     });
 
     // Listen for close events
+    this.$dialog.addEventListener('keydown', (event) => {
+      if (event.key === 'Escape') {
+        this.setDialogState(false);
+      }
+    });
     this.$backdrop.addEventListener('click', () => {
       this.setDialogState(false);
     });
     this.$dialogCloseButton.addEventListener('click', () => {
       this.setDialogState(false);
     });
+
+    // Listen for keyboard events
+    this.$dialog.addEventListener('keydown', (event) => {
+      this.dialogFocusTrap(event);
+    });
+
```

Comment:
What's going to happen here is that even when the element is disconnected, the event listeners will remain, and if reconnected, then duplicate listeners will be registered.  Check out how I registered these in [myuw-profile/pull/27](https://github.com/myuw-web-components/myuw-profile/pull/27), by _declaring_ all the event listeners in the constructor, and then adding/removing them consistently in the connected and disconnected callbacks, respectively.

On a related note, I would move all the _other_ stuff into the constructor